hugo/.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "printWidth": 120,
```

Comment:
Personally, I dislike max line widths quite a bit, especially narrow ones. It see it's at least 120, which could be fine.

Reply:
A maximum of 120 is consistent with how we format our C++ code.
I'd also be fine with let's say 150 but I really think line breaking increases readability as you're not dependent on your editor to do auto line wrapping.